smithy-model/src/test/resources/software/amazon/smithy/model/errorfiles/validators/paginated/paginated-invalid.errors

```diff
@@ -5,7 +5,7 @@
 [ERROR] ns.foo#Invalid3: paginated trait `outputToken` member `OutputNotString` targets a integer shape, but must target one of the following: [`map`, `string`] | PaginatedTrait
 [ERROR] ns.foo#Invalid4: paginated trait `inputToken` member `nextToken` must not be required | PaginatedTrait
 [ERROR] ns.foo#Invalid4: paginated trait `outputToken` member `nextToken` must not be required | PaginatedTrait
-[WARNING] ns.foo#Invalid4: paginated trait `pageSize` member `pageSize` should not be required | PaginatedTrait
+[WARNING] ns.foo#Invalid4: paginated trait `pageSize` member `pageSize` should not be required | PaginatedTrait.pageSize.pageSize.SHOULD_NOT_BE_REQUIRED
```

Comment:
Similar to the other paginated trait validator, I'd like to see the category first, and `pageSize` shouldn't be repeated:
`PaginatedTrait.ShouldNotBeRequired.pageSize`

Reply:
I don't think it appearing twice is useful though and instead you could just always use the terms that are defined by the paginated trait rather than the actual members used in the model (e.g., `items`, `pageSize`, `inputToken`, etc). So that could translate into `PaginatedTrait.ShouldNotBeRequired.pageSize`, which would be emitted event if the member bound to pageSize is MaxResults or something else.

> Are we sure that "PaginatedTrait.ShouldNotBeRequired.[member]" can NEVER appear twice for the same target shape?

This is addressed above.

> What if multiple traits mistakenly require members with the same name?

Not sure I follow, but I suspect this is addressed above.

Reply:
I still am not sure I get it. What if you had a scenario of:

[WARNING] ns.foo#Invalid4: paginated trait `pageSize` member `pageSize` should not be required
and
[WARNING] ns.foo#Invalid4: paginated trait `pageSize` member `someOtherMember` should not be required

Wouldn't those both map to "PaginatedTrait.ShouldNotBeRequired.pageSize"

Reply:
Ah I understand now. I had assumed that these traits could have arbitrarily many members. Will remove the member name 👍